uninstall.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * performance-lab uninstall method
+ */
+
+// if uninstall.php is not called by WordPress, die
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	die;
```

Comment:
Super-nitpick, but I think most plugins use `exit` :)
```suggestion
	exit;
```

Reply:
Resolved, thank you